linux_os/guide/system/permissions/files/permissions_important_account_files/file_owner_backup_etc_shadow/rule.yml

```diff
@@ -0,0 +1,37 @@
+documentation_complete: true
+
+title: 'Verify Group Who Owns Backup shadow File'
+
+description: '{{{ describe_file_group_owner(file="/etc/shadow-", group="root") }}}'
+
+rationale: |-
+    The <tt>/etc/shadow-</tt> file is a backup file of <tt>/etc/shadow</tt>, and as such,
+    it contains the list of local system accounts and password hashes.
+    Protection of this file is critical for system security.
+
+severity: medium
+
+identifiers:
+    cce@rhel7: 83412-7
+    cce@rhel8: 83413-5
+
+references:
+    cis@rhel7: 6.1.7
+    cis@rhel8: 6.1.7
+
+ocil_clause: '{{{ ocil_clause_file_group_owner(file="/etc/shadow-", group="root") }}}'
+
+ocil: '{{{ ocil_file_group_owner(file="/etc/shadow-", group="root") }}}'
+
+template:
+    name: file_groupowner
+    vars:
+        filepath: /etc/shadow-
+        filegid: '0'
+        filegid@debian8: '42'
```

Comment:
I believe that if you make customizations per product, this should be also reflected in description / ocil.

Reply:
The text of rules `file_owner_backup_etc_shadow` and `file_groupowner_backup_etc_shadow` were swapped. Fixed by rebase.